### PR TITLE
Archer integration improvements

### PR DESCRIPTION
**Pull request by hirshbergshachar**
Archer now gets a record's inner records in getRecord & fetch-incidents
fixes https://github.com/demisto/etc/issues/10752

### CODE REVIEW

**Comment by anara123** [comment]
@dorsha what is the status of this PR?
@hirshbergshachar Please delete the `fetchFields` param, we always should return all the fields. And I also think we should remove `fetchByField`, this should be always be Date Created

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -1030,16 +1374,29 @@ script:
         }
     }
 
+    function reLogin(token){
+        destroyAuthToken(token);
+        return getAuthToken();
+    }
+
+    function test() {
```
Comment (anara123) [inline Integrations/integration-ArcherRSA.yml]: You need to do some request, run some command so that the test be reliable

**Comment by hirshbergshachar** [comment]
@anara123

**Comment by anara123** [comment]
It should be merged soon.waiting for buid to be green

**Pull request merged**